google/resource-snippets/pubsub-v1/alternatives/tf/pubsub.tf

```diff
@@ -1,64 +1,40 @@
 provider "google" {
-  project = "{PROJECT_ID}"
+  project = var.project_id
   region  = "{REGION}"
   zone    = "{ZONE}"
 }
 
 variable "deployment" {
- default={}
+  type        = string
+  description = "Deployment name used to label the resources created."
+  default     = {}
 
 }
 
 variable "project_id" {
- default={}
+  type        = string
+  description = "Project id used to create resources in that project."
+  default     = {}
 
 }
 
-resource "google_pubsub_topic" "{TOPIC_REF}" {
+resource "google_pubsub_topic" "my-topic" {
   name = "{TOPIC_NAME}"
```

Comment:
Concrete value. Same for subscription below.

Reply:
@paulmoon are you able to see the changes in my file? My file already has the concrete value, maybe it's not reflecting correctly

Reply:
It changed at 68a4bcaba4798aebce0807c3a37ec25384a02192 which was after my review, so maybe you had it locally but didn't include it in your commits.